ta/pkcs11/src/serializer.c

```diff
@@ -89,3 +90,29 @@ bool serialargs_remaining_bytes(struct serialargs *args)
 {
 	return args->next < args->start + args->size;
 }
+
+/*
+ * Helper as PKCS11_CKR_SESSION_HANDLE_INVALID shall take precedence over
```

Comment:
This is the resumption of https://github.com/OP-TEE/optee_os/pull/3750#discussion_r408701229

If the the serialized arguments cannot be unpacked as expected it would make sense to return `PKCS11_CKR_ARGUMENTS_BAD` instead of trying to make a session of the garbage first.

On the other hand, this function simplify things and the client should not have any detailed expectations on the return code if it feeds garbage.

So if we just drop this about precedence I'm fine with this function.

Reply:
done